DigitalLearningSolutions.Web.AutomatedUiTests/AccessibilityTests/BasicAuthenticatedAccessibilityTests.cs

```diff
@@ -87,30 +90,36 @@ public class BasicAuthenticatedAccessibilityTests : AccessibilityTestsBase,
         )]
         [InlineData("/TrackingSystem/Delegates/3/View", "xxxx xxxxxx")]
         [InlineData("/TrackingSystem/Delegates/3/Edit", "Edit delegate details")]
-        [InlineData("/TrackingSystem/Delegates/3/SetPassword?returnPageQuery=pageNumber%3D1", "Set delegate user password")]
-        [InlineData("/TrackingSystem/Delegates/3/View/100/ViewDelegate/Remove", "Remove enrolment")]
+        [InlineData(
+            "/TrackingSystem/Delegates/3/SetPassword?returnPageQuery=pageNumber%3D1",
+            "Set delegate user password"
+        )]
         [InlineData("/TrackingSystem/Delegates/Approve", "Approve delegate registrations")]
         [InlineData("/TrackingSystem/Delegates/BulkUpload", "Bulk upload/update delegates")]
         [InlineData("/TrackingSystem/Delegates/Email", "Send welcome messages")]
         [InlineData("/TrackingSystem/Delegates/Courses/1", "Delegate courses")]
         [InlineData("/TrackingSystem/Delegates/CourseDelegates", "Course delegates")]
         [InlineData("/TrackingSystem/Delegates/CourseDelegates/DelegateProgress/243104", "Delegate progress")]
         [InlineData(
-            "/TrackingSystem/Delegates/CourseDelegates/DelegateProgress/243104/EditSupervisor",
+            "/TrackingSystem/Delegates/CourseDelegates/DelegateProgress/243104/EditSupervisor?returnPageQuery=pageNumber%3D1",

```

Comment:
I don't understand your logic here. By coming from the course delegates page, there must always be a ReturnPageQuery so that we know where to return to.